src/components/ModalForm.js

```diff
@@ -33,7 +32,7 @@ export default function ModalForm(props) {
   };
 
   return (
-    <div>
+    <>
```

Comment:
do you need an empty tag?

Reply:
turns out I actually didn't need it. A modal acts like a div. 

Removed